runtime/wasm/src/host_exports.rs

```diff
@@ -206,7 +206,7 @@ where
             .find(|abi| abi.name == unresolved_call.contract_name)
             .ok_or_else(|| {
                 HostExportError(format!(
-                    "Unknown contract \"{}\" called from WASM runtime",
+                    "Could not find ABI for contract \"{}\", try adding it to the 'abis' section of the subgraph manifest",
```

Comment:
Let's wrap this string around so the line isn't so long. Other than that, I like this very much. 👍

Reply:
Done